package.json

```diff
@@ -27,15 +27,25 @@
     "grunt-svgmin": "^3.1.2",
     "grunt-usemin": "^3.1.1",
     "grunt-wiredep": "^2.0.0",
+    "jasmine": "^2.4.1",
+    "jasmine-core": "^2.4.1",
     "jshint-stylish": "^2.1.0",
+    "karma": "^1.1.1",
+    "karma-jasmine": "^1.0.2",
+    "karma-junit-reporter": "^1.1.0",
+    "karma-phantomjs-launcher": "^1.0.1",
+    "karma-spec-reporter": "0.0.26",
     "load-grunt-tasks": "^3.4.0",
     "time-grunt": "^1.3.0"
   },
   "engines": {
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "grunt test"
+    "postinstall": "bower install",
+    "pretest": "npm install",
+    "test": "karma start test/karma.conf.js",
+    "test-single-run": "karma start test/karma.conf.js --single-run"
```

Comment:
@msabramo Thank you for this. I totally forgot about `run`. It probably will not pass because in `pretest` you do an `npm install` which I have to do before running the single test as well, right?